src/main/scala/DocGenerator.scala

```diff
@@ -245,7 +251,45 @@ object LatexDocGenerator extends DocGenerator {
                     """
   }
 
+
+  def hypertarget(e: {def id: Seq[Tree]}, text: Option[String]): String = {
+    val t = text.getOrElse("")
+    if (e.id.isEmpty)
+      t
+    else
+      s"\\hypertarget{${link(e.id)}}{$t}"
+  }
+
+  def hyperlink(e: {def id: Seq[Tree]}, text: Option[String]) = {
+    val t = text.getOrElse("")
+    if (e.id.isEmpty)
+      t
+    else
+      s"\\hyperlink{${link(e.id)}}{$t}"
+  }
+
+  def link(tpe: Seq[Tree]) = {
+    val termToTerm = "."
+    val termToType = "."
+    val typeToType = "#"
+    val typeToTerm = "_"
```

Comment:
Edit there in the previous comment for you email users; I'm not sure if it sends you email notifications of edits.